### PR TITLE
Add successful, failure and form validation sign in tests.

### PR DESCRIPTION
**Pull request by germanStabile**
- Fixed some linter warnings

Screen Shot 2020-03-09 at 11 04 57 AM

### CODE REVIEW

File: ios-baseUITests/XCUIApplicationExtension.swift [hunk 1]
```diff
@@ -21,4 +21,35 @@ extension XCUIApplication {
...
+    let logOutButton = buttons["LogoutButton"]
+    let goToSignInButton = buttons["GoToSignInButton"]
+    
+    if logOutButton.exists {
+      logOutButton.forceTap()
+      testCase.waitFor(element: goToSignInButton, timeOut: 5)
+    }
+  }
+  
+  func attemptSignIn(in testCase: XCTestCase,
```
Comment (glm4) [inline ios-baseUITests/XCUIApplicationExtension.swift:35]: shouldn't be each param in its own line?

File: ios-baseUITests/ios_baseUITests.swift [hunk 2]
```diff
@@ -31,28 +33,103 @@ class ios_baseUITests: XCTestCase {
...
+    app.launch()
+    
+    app.logOutIfNeeded(in: self)
+    
+    app.attemptSignIn(in: self,
+                      with: "automation@test.com",
+                      password: "incorrect password")
+    
+    if let alert = app.alerts.allElementsBoundByIndex.first {
+      waitFor(element: alert, timeOut: 2)
```
Comment (glm4) [inline ios-baseUITests/ios_baseUITests.swift:97]: maybe adding a default timeout?

**Comment by germanStabile** [comment]
> Just curious, didn't we have these tests implemented already?

This was a re-implementation with the native framework

File: ios-baseUITests/ios_baseUITests.swift (hunk 2 above)
Comment (germanStabile) [reply ios-baseUITests/ios_baseUITests.swift:97]: I think there should be but I don't recall exactly

**Pull request merged**